scss/atoms/buttons/_buttons.scss

```diff
@@ -39,49 +39,49 @@
 }
 
 .btn {
-	@include btn(	map-get($ui-btn, margin));
```

Comment:
i thought we said it didnt make sense to namespace the variables? if we want to we just need to set the namespace option.